cranelift/filetests/filetests/legalizer/pass_by_ref.clif

```diff
@@ -0,0 +1,35 @@
+test compile
+set opt_level=speed_and_size
+set is_pic
+target x86_64 haswell
```

Comment:
Does this test need all of these things? I would expect

```
test legalizer
target x86_64
```

in the `filetests/legalizer` directory.

It isn't clear to me that we need to go all the way through compilation to exercise the things this test is nominally checking, nor would we need `is_pic` and `opt_level=speed_and_size`. Am I missing something?

Reply:
You're right, I'll change it